evals/registry/evals/recoup.yaml

```diff
@@ -10,3 +10,4 @@ recoup.dev.v0:
     modelgraded_spec: closedqa
     modelgraded_spec_args:
       criteria: "correctness: The answer should be numerically correct and include appropriate units (if applicable). The solution should match exactly with the reference answer."
+      question: Recoup Chat Benchmarks
```

Comment:
hmmmm, I'm not sure `prompt` or `question` mkes sense here as this text is not being passed as a prompt but rather it is a `label` for a group of evals.

What do you think of the name `label` for this variable?